astroquery/ipac/irsa/core.py

```diff
@@ -122,6 +128,42 @@ def query_sia(self, *, pos=None, band=None, time=None, pol=None,
 
     query_sia.__doc__ = query_sia.__doc__.replace('_SIA2_PARAMETERS', SIA2_PARAMETERS_DESC)
 
+    def query_ssa(self, *, pos=None, diameter=None, band=None, time=None, format=None,
+                  collection=None):
+        """
+        Use standard SSA attributes to query the IRSA SSA service.
+
+        Parameters
+        ----------
+        pos : `~astropy.coordinates.SkyCoord` class or sequence of two floats
+            the position of the center of the circular search region.
+            assuming icrs decimal degrees if unit is not specified.
+        diameter : `~astropy.units.Quantity` class or scalar float
+            the diameter of the circular region around pos in which to search.
+            assuming icrs decimal degrees if unit is not specified.
+        band : `~astropy.units.Quantity` class or sequence of two floats
+            the bandwidth range the observations belong to.
+            assuming meters if unit is not specified.
+        time : `~astropy.time.Time` class or sequence of two strings
+            the datetime range the observations were made in.
+            assuming iso 8601 if format is not specified.
+        collection : str
+           Name of the collection that the data belongs to.
+        format : str
+           the image format(s) of interest.  "all" indicates
+           all available formats; "graphic" indicates
+           graphical images (e.g. jpeg, png, gif; not FITS);
+           "metadata" indicates that no images should be
+           returned--only an empty table with complete metadata.
```

Comment:
is `('all', 'graphic', 'metadata')` a complete set of valid options?  If so, it would be helpful to indicate that.  If not, can we document what is allowed here or in the main docs?

Reply:
I copied this from the pyvo docs, looking at the IRSA SSA metadata, these are the actual options:

```
        <OPTION value="ALL"/>
        <OPTION value="COMPLIANT"/>
        <OPTION value="NATIVE"/>
        <OPTION value="VOTABLE"/>
        <OPTION value="FITS"/>
        <OPTION value="XML"/>
        <OPTION value="GRAPHIC"/>
        <OPTION value="METADATA"/>
```

However, some of these are known not to be working (e.g. metadata), and now as I test them some of them are not working as expected. (e.g. FITS). 

So this may need an upstream fix, and here, I may just remove this kwarg for now and hard-wire to return all results in this first iteration.

Reply:
so, now I hardwired `'all'`, and opened a follow-up to address this properly. xref: #3234